plutus-core/executables/debugger/Event.hs

```diff
@@ -74,7 +75,7 @@ handleDebuggerEvent driverMailbox bev@(B.VtyEvent ev) = do
             -- This disables editing the text, making the editors read-only.
             pure ()
         _ -> handleEditorEvent
-handleDebuggerEvent _driverMailbox (B.AppEvent (UpdateClientEvent cekState)) = do
+handleDebuggerEvent _driverMailbox (B.AppEvent (UpdateClientEvent BudgetData{..} cekState)) = do
```

Comment:
I usually avoid ay Record* language extensions because i find it hard to remember their syntax :D . I thought to give them a try since some other parts of the debugger use them. I can refactor them out